src/main/webapp/script.js

```diff
@@ -103,3 +102,39 @@ function onLoad() {
   getDatabases();
   getTablesList();
 }
+
+function showShare() {
+    console.log("Show");
+    document.getElementById("share-form").classList.remove("invisible");
+}
+
+function copyLink() {
+    var copyText = document.getElementById("share-text");
+    copyText.select();
+    copyText.setSelectionRange(0, 99999)
+    document.execCommand("copy");
+    alert("Copied the text: " + copyText.value);
+}
+
+var previousText = "placeholder";
+var previousId = "id";
+
+function luckyFilter() {
+    previousId="filter-button";
+    previousText = document.getElementById("filter-button").textContent;
+    document.getElementById("filter-button").textContent = "I'm Feeling Lucky!";
+}
+
+function luckyData() {
+    previousId="data-button";
+    previousText = document.getElementById("data-button").textContent;
+    document.getElementById("data-button").textContent = "I'm Feeling Lucky!";
+}
+
+function unlucky() {
+    document.getElementById(previousId).textContent = previousText;
+}
```

Comment:
Are these for the easter egg? If not maybe rename to fit the context more (I might just be confused)

Reply:
Yeah these are more of an idea of the easter egg from what Guillermo mentioned about hovering over the button and the name changing! I should have clarify in a comment the reason for the bizarre naming.